apps/mobile/src/screens/Stream/Stream.js

```diff
@@ -194,6 +192,7 @@ export default function Stream () {
     updateUserSettings({ changes: { settings: { streamChildPosts: childPostInclusion } } })
   }
 
+  // TODO: Unused. Remove or explain in comment
```

Comment:
Ahh interesting. This was for a stream control toggle. Looks like its being handled elsewhere now, since it works as expected on dev. So we can nuke this.

Reply:
Ok, next time you see it do that, otherwise I will try and remember when I see it next :)